flybywiresim/aircraft
# fix(system): origin/dest not reported to API

Fixes #[issue_no]

## Summary of Changes


The aircraft was not reporting the ORIGIN/DESTINATION set in the flight plan to the API so I fixed it :)

## Screenshots (if necessary)



![image](https://github.com/flybywiresim/a32nx/assets/18389085/f389f427-da60-4a12-8b6e-9b595ee7c7a7)
![image](https://github.com/flybywiresim/a32nx/assets/18389085/296839af-633b-4a81-b376-58a4b7b217f3)


![image](https://github.com/flybywiresim/a32nx/assets/18389085/5b416ff2-519e-4c50-9673-3ff9371ad9fe)


## References





N/A
## Additional context



Discord username (if different from GitHub):
NathanI#8668
## Testing instructions


1. Start up the aircraft with power, populate the INIT page of MCDU with information (see below): 

![image](https://github.com/flybywiresim/a32nx/assets/18389085/0630501c-4ba4-41d1-8b25-54ea5563ed62)

2. Visit https://api.flybywiresim.com/api/#/TELEX/TelexConnectionController_findConnection and put the callsign into the box and press execute.
3. Observe the section below and ensure that the response has information

![image](https://github.com/flybywiresim/a32nx/assets/18389085/1f67147c-51a1-4c27-90e6-061dd98ea109)

**NOTE: Ensure TELEX is enabled for this to work.**

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
